test/features/resource_availability.feature

```diff
@@ -1,252 +1,113 @@
-        Then I should see "Re-identification risk governance completed?" 
-        Then I log out 
-        And "TestOrgEditor" as persona
-        And I log in
-        And I go to "/contains-de-identified-data-yes-visibility-true-acknowledged-yes"
-        And I wait for 3 seconds
-        Then I should see element with xpath "//a[@title='Resource not hidden']"
-        Then I press the element with xpath "//a[@title='Resource not hidden']"
-        Then I should see "Resource visible"
-        Then I should see "Re-identification risk governance completed?" 
-        Then I log out 
-        And "User" as persona
-        And I go to "/contains-de-identified-data-yes-visibility-true-acknowledged-yes"
-        And I wait for 3 seconds
-        Then I should see element with xpath "//a[@title='Resource not hidden']"
-        Then I press the element with xpath "//a[@title='Resource not hidden']"
-        Then I should not see "Resource visible"
-        Then I should not see "Re-identification risk governance completed?" 
-
-
-Scenario Outline: Non logged in users see the resource when de-identified data is YES and Resource visibility is TRUE and Acknowledgement is YES
-        Given "TestOrgAdmin" as the persona
-        And I log in
-        And I go to "/contains-de-identified-data-yes-visibility-true-acknowledged-yes"
-        And I wait for 3 seconds
-        Then I should see element with xpath "//a[@title='Resource not hidden']"
-        Then I press the element with xpath "//a[@title='Resource not hidden']"
-        Then I should see "Resource visible"
-        Then I should see "Re-identification risk governance completed?" 
-        Then I log out 
-        And "TestOrgEditor" as persona
-        And I log in
-        And I go to "/contains-de-identified-data-yes-visibility-true-acknowledged-yes"
-        And I wait for 3 seconds
-        Then I should see element with xpath "//a[@title='Resource not hidden']"
-        Then I press the element with xpath "//a[@title='Resource not hidden']"
-        Then I should see "Resource visible"
-        Then I should see "Re-identification risk governance completed?" 
-        Then I log out 
-        And "User" as persona
-        And I go to "/contains-de-identified-data-yes-visibility-true-acknowledged-yes"
-        And I wait for 3 seconds
-        Then I should see element with xpath "//a[@title='Resource not hidden']"
-        Then I press the element with xpath "//a[@title='Resource not hidden']"
-        Then I should not see "Resource visible"
-        Then I should not see "Re-identification risk governance completed?" 
-
-
-    ###
-    # Create resource  with default values
-    ###
-    Scenario Outline: Create resource with default values
+
+    Scenario: Unauthenticated user should see a visible resource when de-identified data is NO and Resource visibility is TRUE and Acknowledgement is NO
+        Given "Unauthenticated" as the persona
```

Comment:
No real need for a persona if we aren't logging in.

Reply:
Yeh that is true, think this was a copy and paste from other tests and I think it was added originally to make it a little bit clearer it's using an unauthenticated person.
I have removed those steps